package.json

```diff
@@ -8,6 +8,8 @@
   "main": "./source/index.js",
   "bin": "./source/index.js",
   "scripts": {
+    "dev-init": "npx react-native init example",
+    "dev-sync": "cp -rv package.json source react-native.config.js example/node_modules/react-native-clean-project/",
```

Comment:
Even with the automated testing, I took tremendous comfort in having quick access to a rebuildable example project with quick install of local module changes, so I left the example items in (here, .gitignore, and ignored in jest config). If you feel strongly that you don't like them, I can drop them